setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
If there's no practical reason to prefer one or the other, I prefer `scitools-iris`.